docs/architecture/incremental.md

```diff
@@ -0,0 +1,66 @@
+# Incremental
+
+## Overview
+
+Incremental syncs in Airbyte allow sources to replicate only new or modified data. This prevents re-sending data that has already been sent to your warehouse. We will call this set of new or updated records the delta going forward.
```

Comment:
Should emphasize that this doesn't pull the full data from the source either.

Reply:
fixed.